src/DetailsView/Styles/detailsview.scss

```diff
@@ -217,6 +217,9 @@ div.insights-file-issue-details-dialog-container {
 
 .failure-instance-panel {
     .observed-failure-textfield {
+        .ms-Label {
```

Comment:
Here and the other case below:

can we figure out a way to get this resulting styling without using "ms-Label" as the class?

Reply:
Changed it to ":first-child".... let me know if this is any better than using "ms-Label"!

Reply:
Changing to using CSS styling functions found on the MS Fabric site which tell us how to safely do this---In the next PR